apps/discord-bot/src/commands/leaderboards/base.leaderboard-command.tsx

```diff
@@ -279,7 +288,8 @@ export class BaseLeaderboardCommand {
     field: string,
     params: LeaderboardParams,
     props: BaseLeaderboardProps,
-    getLeaderboardDataIcon?: GetLeaderboardDataIcon
+    getLeaderboardDataIcon?: GetLeaderboardDataIcon,
+    time?: HistoricalType | undefined
```

Comment:
```suggestion
    time?: HistoricalType
```

I would also use the new enum like I suggested above